static/app/utils/fields/index.ts

```diff
@@ -2876,7 +2876,14 @@ const FEEDBACK_FIELD_DEFINITIONS: Record<FeedbackFieldKey, FieldDefinition> = {
 
 export const getFieldDefinition = (
   key: string,
-  type: 'event' | 'replay' | 'replay_click' | 'feedback' | 'span' | 'log' = 'event',
+  type:
+    | 'event'
+    | 'replay'
+    | 'replay_click'
+    | 'feedback'
+    | 'span'
+    | 'log'
+    | 'uptime' = 'event',
```

Comment:
Nit: This is for the search bar options, not really relevant to the trace waterfall. I think we should remove this for now.

Reply:
There was some reason I needed to add this here to make types happy